dist/images/ovnkube.sh

```diff
@@ -3,70 +3,59 @@
 #set -euo pipefail
 
 # This script is the entrypoint to the image.
-# Supports version 1, 2, and 3 daemonsets
-#    $1 is the daemon to start. In version 2 the daemonset has a container for
-#        each daemon. This start script is called with the desired daemon. In
-#        version 3 each process has a separate container. Some daemons start
+# Supports version 3 daemonsets
+#    $1 is the daemon to start.
+#        In version 3 each process has a separate container. Some daemons start
 #        more than 1 process. Also, where possible, output is to stdout and
 #        The script waits for prerquisite deamons to come up first.
-#        The default, with $1 == "" is to start all daemons one after the other
-#        waiting for a daemon to start before starting the next (version 1).
 # Commands ($1 values)
-#    ovs-server     Runs the ovs daemons - ovsdb-server and ovs-switchd (v2, v3)
+#    ovs-server     Runs the ovs daemons - ovsdb-server and ovs-switchd (v3)
 #    run-ovn-northd Runs ovn-northd as a process does not run nb_ovsdb or sb_ovsdb (v3)
 #    nb-ovsdb       Runs nb_ovsdb as a process (no detach or monitor) (v3)
 #    sb-ovsdb       Runs sb_ovsdb as a process (no detach or monitor) (v3)
-#    ovn-northd     Runs ovn-northd daemon - (v2)
-#    ovn-master     Runs ovnkube in master mode (v2, v3)
-#    ovn-controller Runs ovn controller (v2, v3)
-#    ovn-node       Runs ovnkube in node mode (v2, v3)
+#    ovn-master     Runs ovnkube in master mode (v3)
+#    ovn-controller Runs ovn controller (v3)
+#    ovn-node       Runs ovnkube in node mode (v3)
 #    cleanup-ovn-node   Runs ovnkube to cleanup the node (v3)
-#    cleanup-ovs-server Cleanup ovs-server
+#    cleanup-ovs-server Cleanup ovs-server (v3)
 #
 #    display        Displays log files
 #    display_env    Displays environment variables
 #    ovn_debug      Displays ovn/ovs configuration and flows
-#
-#    <no argumet>   Runs all daemons - version 1 compatibility
 
 # NOTE: The script/image must be compatible with the daemonset.
-# This script supports version 1, 2, 3 daemonsets
+# This script supports version 3 daemonsets
 #      When called, it starts all needed daemons.
 
 # ====================
 # Environment variables are used to customize operation
-# K8S_APISERVER - hostname:port (URL)of the real apiserver, not the service address
-# OVN_NET_CIDR - the network cidr
-# OVN_SVC_CIDR - the cluster-service-cidr
-# OVN_KUBERNETES_NAMESPACE - k8s namespace
+# K8S_APISERVER - hostname:port (URL)of the real apiserver, not the service address - v3
+# OVN_NET_CIDR - the network cidr - v3
+# OVN_SVC_CIDR - the cluster-service-cidr - v3
+# OVN_KUBERNETES_NAMESPACE - k8s namespace - v3
+# K8S_NODE - hostname of the node - v3
 #
 # The following variables are optional and can override internally derived values.
 # OVN_NORTH - the full URL to the ovn northdb
 # OVN_SOUTH - the full URL to the ovn southdb
 #
```

Comment:
OVN_NORTH and OVN_SOUTH are not needed, so you can remove it. You cannot override the OVN DB TCP endpoint using OVN_NORTH and OVN_SOUTH anymore since we use ovnkube-db kubernetes service .

Reply:
@girishmg so ovnkube --nb-address and --sb-address must not be used?
I will cleanup all related areas.

Reply:
@pecameron I am saying that the value for the CLI option --nb-address and --sb-address is derived from the ovnkube-db K8s service's endpoint IP. You cannot override it by exporting environment variables OVN_NORTH and OVN_SOUTH.

In other words,  in the NB TCP connection `tcp:<ip_address>:6641`, the **<ip_address>** is derived from the service's endpoint. You cannot override that value using OVN_NORTH environment variable anymore.

The following
```
# The following variables are optional and can override internally derived values.
# OVN_NORTH - the full URL to the ovn northdb
# OVN_SOUTH - the full URL to the ovn southdb
```
isn't true. So, you need to just remove those environment variables. So,

```
get_ovn_db_vars () {
  # OVN_NORTH and OVN_SOUTH override derived host
  # Currently limited to tcp (ssl is not supported yet)
  ovn_nbdb=${OVN_NORTH:-tcp://${ovn_db_host}:6641}
  ovn_sbdb=${OVN_SOUTH:-tcp://${ovn_db_host}:6642}
  ovn_nbdb_test=$(echo ${ovn_nbdb} | sed 's;//;;')
}
```

will become

```
get_ovn_db_vars () {
  ovn_nbdb=tcp://${ovn_db_host}:6641
  ovn_sbdb=tcp://${ovn_db_host}:66421
  ovn_nbdb_test=$(echo ${ovn_nbdb} | sed 's;//;;')
}
```

Reply:
@pecameron  can you please see my comment above?

In your latest changes you have removed --nb-address and --sb-address. That will not work. Can you please revert it? Also, keep the get_ovn_db_vars () and the associated ovn_nbdb and ovn_sbdb variables.

Reply:
Done